transmart-core-api/src/main/groovy/org/transmartproject/core/patient/anonomization/AbstractThresholdPatientDataAnonymizer.groovy

```diff
@@ -0,0 +1,28 @@
+package org.transmartproject.core.patient.anonomization
+
+import org.transmartproject.core.multidimquery.counts.Counts
+
+/**
+ * Use threshold below which does not show patient data.
+ */
+abstract class AbstractThresholdPatientDataAnonymizer implements PatientDataAnonymizer {
```

Comment:
What's the point of the interface and the abstract class? It does not make the code clearer.